server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxGetMethodContract.scala

```diff
@@ -162,4 +269,444 @@ trait MailboxGetMethodContract {
         |            "c1"]]
         |}""".stripMargin)
   }
+
+  @Test
+  def getMailboxesShouldReturnSharedRights(server: GuiceJamesServer): Unit = {
```

Comment:
Do we have a test to show that as a sharee I see my rights but not other people rights?